<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TamdidBookByUser</class>
 <widget class="QDialog" name="TamdidBookByUser">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1282</width>
    <height>847</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QTableWidget" name="tableWidget">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>100</y>
     <width>711</width>
     <height>541</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: qconicalgradient(cx:0.5, cy:0.5, angle:0, stop:0 rgba(250, 73, 125, 87), stop:0.151685 rgba(250, 17, 17, 98), stop:0.339799 rgba(255, 187, 187, 130), stop:0.5 rgba(255, 60, 60, 87), stop:0.668539 rgba(255, 0, 0, 89), stop:0.848315 rgba(234, 86, 86, 28), stop:1 rgba(243, 32, 32, 102));</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>710</y>
     <width>111</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>IranNastaliq</family>
     <pointsize>22</pointsize>
    </font>
   </property>
   <property name="text">
    <string>بازگشت</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>900</x>
     <y>700</y>
     <width>111</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>IranNastaliq</family>
     <pointsize>22</pointsize>
    </font>
   </property>
   <property name="text">
    <string>اعمال تغییرات</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
